src/main/java/ch/heig/gen/MainCommand.java

```diff
@@ -5,7 +5,7 @@
 @CommandLine.Command(
         name = "arancia",
         subcommands = {NewCommand.class, CleanCommand.class, BuildCommand.class, ServeCommand.class},
-        description = "Description stub.",
+        description = "Program allowing the generation of static site from a simple Markdown document\n",
```

Comment:
The program does not 'allow' anything (since it's not forbidden or otherwise restricted).

```suggestion
        description = "Generates static websites from Markdown documents\n",
```